Misc/gdbinit

```diff
@@ -54,25 +54,7 @@ end
 # A rewrite of the Python interpreter's line number calculator in GDB's
 # command language
 define lineno
-    set $__continue = 1
-    set $__co = f->f_code
-    set $__lasti = f->f_lasti
-    set $__sz = ((PyVarObject *)$__co->co_lnotab)->ob_size/2
-    set $__p = (unsigned char *)((PyBytesObject *)$__co->co_lnotab)->ob_sval
-    set $__li = $__co->co_firstlineno
-    set $__ad = 0
-    while ($__sz-1 >= 0 && $__continue)
-      set $__sz = $__sz - 1
-      set $__ad = $__ad + *$__p
-      set $__p = $__p + 1
-      if ($__ad > $__lasti)
-	set $__continue = 0
-      else
-        set $__li = $__li + *$__p
-        set $__p = $__p + 1
-      end
-    end
-    printf "%d", $__li
+    printf "%d", f->f_lineno
```

Comment:
f_lineno is only defined when it is non-zero, so this logic will need to be re-implemented for the new line number table.
Not a fun task, sorry.

Does this file have any tests? I suspect that most of the code in this file is wrong (as we have changed much of CPython since it was written).